test-.tf

```diff
@@ -0,0 +1,11 @@
+resource "azurerm_postgresql_server" "example_ssl_enforcement_enabled" {
+    name                                = "example-psqlserver"
+    location                            = azurerm_resource_group.example.location
+    resource_group_name                 = azurerm_resource_group.example.name
+    administrator_login                 = "psqladminun"
+    administrator_login_password        = "H@Sh1CoR3!"
+    sku_name                            = "GP_Gen5_4"
+    storage_mb                          = 640000
+    version                             = "9.6"
+    ssl_enforcement_enabled             = false
```

Comment:
:exclamation:**Cycode**: Infrastructure configuration issue: 'Ensure 'Enforce SSL connection' is set to 'ENABLED' for PostgreSQL Database Server'.
Severity: High

Would you like to exclude this IaC violation from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_iac_false_positive** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_iac_fix_this_violation** | Fix this violation via a commit to this branch |
| **#cycode_ignore_violation_for_resource_everywhere** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_resource_everywhere** | Applies to this resource for all repos in your organization |
| **#cycode_ignore_violation_for_resource_here** | Applies to this resource for this violation in this request only |

 <!-- 51a3692c-ea4b-4672-a870-07da8107d66c -->